### DISCUSSION TITLE
The education mailbox cannot accept GitHub's verification email

### DISCUSSION THREAD

**Discussion by Leokiko** [community/community#51431] [GitHub Education]
After binding the education mailbox on github, click the "Resend verification email" button, but the verification email has not been received. So I changed to my Gmail mailbox and Outlook mailbox, and I can receive emails successfully. And I also consulted the email provider, they say the mailbox server settings are all right. I suspect it may be because the email domain is invalid for GitHub, please help me!

The email domain is @mails.swust.edu.cn

**Comment by samus-aran** [comment]
Hi @Leokiko 👋 

Depending on the school you are applying for an academic email may be required. We unfortunately can not accommodate for an institute that has restrictions on receiving external mail. We do allow non academic emails to be submitted but it is preferred to have a verified academic email.

**Reply by Leokiko** [reply]
First of all thank you very much for your help!

Maybe you misunderstood me, my academic email does accept external emails, I have tested that it can receive emails from Gmail and Outlook and can also reply to these emails.

![picture](https://user-images.githubusercontent.com/31812456/229000031-d66e56a3-32f3-4d3e-9723-fa6710fd5376.png)
You can see my sending record information, indicating that GitHub's email server is reachable, but I cannot receive emails from GitHub.

![pic](https://user-images.githubusercontent.com/31812456/229002382-86d0593e-0b20-4bd5-8655-02a7f326afcf.jpg)
At the same time, I found that when I applied for Unidays on apple.com, my email domain was marked as unverified. So I think the problem may be in GitHub's policy on my email domain, maybe GitHub also set my email domain to an unverified domain?


Is there a way to solve it?

Thank you again!
